code/radar/radar.cpp

```diff
@@ -359,6 +359,27 @@ void HudGaugeRadarStd::pageIn()
 	bm_page_in_aabitmap( Radar_gauge.first_frame, Radar_gauge.num_frames );
 }
 
+void HudGaugeRadarStd::clampBlip(vec3d* blip)
```

Comment:
"blip" is a struct defined in radarsetup, gcc and clang both think that choosing a different name for this parameter would be good (Or, alternatively, you could pass in the actual blip here?)